server/src/main/java/org/apache/druid/server/http/CoordinatorCompactionConfigsResource.java

```diff
@@ -204,8 +213,9 @@ Response updateConfigHelper(Callable<SetResult> updateMethod)
       }
     }
     catch (Exception e) {
+      String errorMessage = e.getMessage();
       return Response.status(Response.Status.INTERNAL_SERVER_ERROR)
-                     .entity(ImmutableMap.of("error", e))
+                     .entity(ImmutableMap.of("error", errorMessage == null ? "Unknown Error" : errorMessage))
```

Comment:
nit: worth making a method to make an error response perhaps

Reply:
Done